### PR TITLE
Windows: Support packaging of external extensions by manifest file

### PR DESCRIPTION
**Pull request by DonnaWuDongxia**
Combine all the extensions into "xwalk-extensions" sub-directory
of the packaged application.

BUG=XWALK-4851

### CODE REVIEW

**Comment by DonnaWuDongxia** [comment]
@r0b5t4 Whould you please help to review this PR? Thanks!

**Comment by DonnaWuDongxia** [comment]
@halton Please help to review. Thanks!

**Comment by DonnaWuDongxia** [comment]
Thanks @halton, more comments added.

**Comment by halton** [comment]
LGTM, let's wait @r0b5t4 back.

File: windows/lib/WinPlatform.js [hunk 1]
```diff
@@ -254,9 +254,10 @@ function(configId, args, callback) {
         version: manifest.appVersion + versionPadding,
         is_64_bit: true,
         icon: this.selectIcon(),
-        product: manifest.packageId
-        //extensions: 'tests/extension/echo_extension'
+        product: manifest.packageId,
+        extensions: manifest.extensions
```
Comment (r0b5t4) [inline windows/lib/WinPlatform.js]: You can access the manifest inside WixSDK by using this.manifest, passing it like this is deprecated.

**Comment by r0b5t4** [comment]
@DonnaWuDongxia patch looks good if the comment above is addressed. Will try to test ASAP. Many thanks.

**Comment by DonnaWuDongxia** [comment]
Updated, @r0b5t4 Please help to review.

**Comment by DonnaWuDongxia** [comment]
Ping @r0b5t4 , any comments?

**Pull request merged**